docs/rst/manual/theory/ensemble_based_methods.rst

```diff
@@ -397,9 +397,97 @@ Iterative Ensemble Smoother - Ensemble subspace version
+
+.. math::
+   \begin{align}
+   \hat{\mu}_{\psi|d} &= \bar{\psi} + \hat{K}(\bar{d}-M\bar{\psi}),\\
+   \hat{\Sigma}_{\psi|d} &= (I-\hat{K}M) \hat{\Sigma}_{\psi}
+   \end{align}
+
+From this, it is seen that when the sample size tends to infinity and estimates converge to the corresponding population quantities,
+then the ensemble variants converge to the standard Kalman filter in the linear Gaussian case.
+This convergence is however of a stochastic nature.
+
+More deterministic properties of the posterior are observed when the belief in measurements :math:`d` is varied. 
+Intuitively, when measurements have zero belief, i.e. unbounded variance, then the posterior should equal the prior.
+At the other end of the spectrum, if the measurements are perfect with zero variance,
+then the posterior estimate should equal the maximum-likelihood estimate, corresponding to a flat prior, 
+and as we are certain of the belief in this estimate (because the measurements are so amazing), 
+the determinant of the posterior covariance tends to zero from above.
+The maximum likelihood estimate is found by minimizing the relevant part of the negative log-likelihood of the data
+
+.. math::
+   \begin{align}
+   \hat{\mu}_{ml} = \arg\min_{\mu} |d-M\mu|_2
+   \end{align}
+
+Furthermore, for a monotone sequence in belief in measurements, a monotone sequence
+for the posterior towards the likelihood estimate, and from the prior, should be observed.
+To summarize: 
+
+- For the posterior estimate, we require that
+  
+  a. The information in :math:`d` has been assimilated, creating a better estimate, so that :math:`|\hat{\mu}_{\psi|d}-\hat{\mu}_{ml}|_2<|\hat{\mu}_{\psi}-\hat{\mu}_{ml}|_2` and :math:`|\hat{\mu}_{\psi|d}-\hat{\psi}_{\psi}|_2<|\hat{\mu}_{\psi}-\hat{\mu}_{ml}|_2`.
+  b. The estimate improves at better quality data: Let :math:`\Sigma_d=\sigma_d I`. If a sequence of :math:`\sigma_d` decreases strictly, then so will the corresponding sequence of :math:`|\hat{\mu}_{\psi|d}-\hat{\mu}_{ml}|_2`.
+  c. The estimate does not move from the prior at no information: When :math:`\sigma_d\to \infty` then :math:`|\hat{\mu}_{\psi|d}-\hat{\mu}_{\psi}|_2\to 0`.
+  d. The estimate sequence converges to the ml-estimate:  When :math:`\sigma_d\to 0` then :math:`|\hat{\mu}_{\psi|d}-\hat{\mu}_{ml}|_2\to 0`.
+
+- For the posterior covariance, we require that
 
+  a. We become more certain of our estimates as informative data is assimilated, thus :math:`0<\det(\Sigma_{\psi|d})<\det(\Sigma_{\psi})`.
+  b. We become increasingly certain in our estimates when increasingly informative data is assimilated: When a sequence of :math:`\sigma_d` decreases strictly, then so will the corresponding sequence of :math:`\det(\Sigma_{\psi|d})`.
+  c. The certainty of our estimate does not move from the priors, when assimilated data contains no information: When :math:`\sigma_d\to \infty` then :math:`\det(\Sigma_{\psi|d})\to\det(\Sigma_{\psi})` from below.
```

Comment:
I think comma before "when" can be removed to improve flow.
Yeah I know, overly pedantic :D

Reply:
Pedantic is good, thanks!

Reply:
Forgot to update this one?

Reply:
😬